add-domain

```diff
@@ -999,3 +999,13 @@ wiflix.xyz
 wiflix.yachts
 coronavirus.zone
 fullcopec.zone
+manta-network-v2.us
+tronilnk.com
+en-trezor.io
+trust.wallet-web3.ing
+coinomi.ing
+defii-larna.net
+app.deflliama.tech
```

Comment:
This one are no longer active and should therefor not be added 

- https://github.com/mypdns/matrix/issues/127013